### PR TITLE
Read The Docs

### PR DESCRIPTION
**Pull request by rgrinberg**
Port manual to RST to make it usable on readthedocs.

conf.py was only there to make it easy to preview changes if you have sphinx
installed.

Sorry I let hang for so long. I just couldn't organize in a way that I 100%
like. I still think it's an improvement. Especially that the most important
section, the jbuild specification, is more prominent.

If this is accepted I'll start working on polishing it.

### CODE REVIEW

**Comment by rgrinberg** [comment]
Also, I've yet to delete the manual.org. Perhaps it's still useful for some of the WIP

**Comment by rgrinberg** [comment]
Once this PR is merged, I expect the docs to show up here: http://jbuilder.readthedocs.io/en/latest/

To preview the docs locally if you have sphinx installed, the following command in the doc directory will work:

```
$ sphinx-build . _build
```

The docs will show up in _build/index.html

You can preview the docs before merging here:
http://testrtdxyz.readthedocs.io/en/latest/

**Comment by rgrinberg** [comment]
readthedocs version of sphinx seems not to like utf8 in the source so I had to mangle your name @diml. Sorry about that.

**Comment by yminsky** [comment]
Have you considered using pandoc to convert to rst instead of changing the
format in the repo?

**Comment by rgrinberg** [comment]
Yes. In fact, that was the starting point for my port. It required lots of
manual touch ups to make it look good however.

**Comment by diml** [comment]
Thanks,  looks good! Happy to move in this direction.

@yminsky, are you thinking that the doc won't look good on github? I think updating the link in the readme and adding a pointer to readthedocs at the beginning of the manual should be enough.

For the utf8 problems, did you try using `u'...'`? Not a blocker but it's shocking if they don't handle unicode...

Two things i noted while looking at the preview:
- the link to the examples in the quickstart is broken
- the code formatting in titles should be removed, it doesn't look good

**Comment by rgrinberg** [comment]
@diml all fixed.

File: doc/conf.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+author = u'Jérémie Dimino'
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+version = '1.0'
+# The full version, including alpha/beta/rc tags.
+release = '1.0'
```
Comment (diml) [inline doc/conf.py]: Could this be computed? For instance using `git tags --describe`

Comment (rgrinberg) [inline doc/conf.py]: I think I can just get rid of it. I'm pretty sure rtd infers the version from git tags anyway

Comment (rgrinberg) [inline doc/conf.py]: http://docs.readthedocs.io/en/latest/versions.html

**Comment by diml** [comment]
Ok, this looks good to me. Feel free to merge

**Comment by rgrinberg** [comment]
OK. So I take it you'll delete manual.org and quickstart.org at a later commit?

**Comment by diml** [comment]
You can delete them in this PR

**Pull request merged**